config/kubernetes-sigs/sig-scheduling/teams.yaml

```diff
@@ -45,6 +45,7 @@ teams:
     members:
     - kerthcet
     - sanposhiho
+    - codefromthecrypt
```

Comment:
This list needs to be sorted.

```
config_test.go:272: The team kube-scheduler-wasm-extension-admins in org Kubernetes SIGs has an unsorted list of members
```

Reply:
Nah. Sorry about taking your time on my overlooking the message. 🙏